R/resampleClusteredRegions.R

```diff
@@ -61,40 +61,40 @@ resampleClusteredRegions <- function(voxel_df, n_pca = 20,
     stop("n_umap and n_pca must be greater than 1")
   }
   else{
-  if (subsamp_prop == 1){
+  if (subsamp_prop == 1 || subsamp_prop == 0){
     stop("subsamp_prop must be less than 1")
   }
   else{
 
   # find the number of columns needed to sample from subsamp_prop
-  num <- floor(ncol(voxel_df[,4:ncol(voxel_df)])*subsamp_prop)
+  num <- floor(ncol(voxel_df[, 4:ncol(voxel_df)])*subsamp_prop)
 
   # create empty data frame which clusters will be stored in later
   cluster_vec <- data.frame(matrix(NA,
                                    nrow = nrow(voxel_df),
                                    ncol = n_resamp))
 
-  voxel_df_data <- voxel_df[,4:ncol(voxel_df)]
+  voxel_df_data <- voxel_df[, 4:ncol(voxel_df)]
 
   # extract xyz coords
-  xyz <- voxel_df[,1:3]
+  xyz <- voxel_df[, 1:3]
 
   # run a for loop to resample and calculate clusters
-  for (i in 1:n_resamp){
+  for (i in 1:n_resamp) {
```

Comment:
Let's print a message either after completing iteration `i` or after starting iteration `i`. We can use the `message` function. We should also message after finishing any relevant steps. Perhaps we can make a `verbose` argument in the function with default `TRUE`